src/dragDrop.js

```diff
@@ -1,22 +1,16 @@
 // Initials tasks
-/* eslint-disable import/no-mutable-exports */
-export let tasks = [
-  {
-    index: 1,
-    description: 'brush my teeth',
-    completed: true,
-  },
-  {
-    index: 2,
-    description: 'Go to school',
-    completed: false,
-  },
-  {
-    index: 3,
-    description: 'Sleep',
-    completed: false,
-  },
-];
+/* eslint-disable import/no-mutable-exports,  */
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-loop-func */
```

Comment:
[OPTIONAL]

- I would recommend instead of disabling the linters you can implement best practices in your code. These linters is for you to make your code better.

Reply:
Yes, this is true, I'm trying to understand those types of errors but since I'm late today please consider this.